CreateWebSource.java

```diff
@@ -19,8 +19,7 @@ public static void main(String[] args) throws Exception
   {
     System.out.println( "" );
```

Comment:
What this is for?

Reply:
This has been came along with code from beginning. Let me remove it.